Dnn.AdminExperience/Library/Dnn.PersonaBar.UI/Services/ComponentsController.cs

```diff
@@ -104,17 +106,27 @@ public HttpResponseMessage GetSuggestionRoles(string keyword, int roleGroupId, i
                     return this.Request.CreateResponse(HttpStatusCode.OK, new List<SuggestionDto>());
                 }
 
-                var matchedRoles = RoleController.Instance.GetRoles(this.PortalId)
-                    .Where(r => (roleGroupId == -2 || r.RoleGroupID == roleGroupId)
-                                && r.RoleName.IndexOf(keyword, StringComparison.InvariantCultureIgnoreCase) > -1
-                                   && r.Status == RoleStatus.Approved)
-                    .Select(r => new SuggestionDto()
+                var portalRoles = RoleController.Instance.GetRoles(this.PortalId);
+                
+
+                var matchedRoles = portalRoles.Where(r => (roleGroupId == -2 || r.RoleGroupID == roleGroupId)
+                                                          && r.RoleName.IndexOf(keyword, StringComparison.InvariantCultureIgnoreCase) > Null.NullInteger
```

Comment:
I thinks this should stay as `-1` since it's a value from .NET Framework and not from DNN
```suggestion
                                                          && r.RoleName.IndexOf(keyword, StringComparison.InvariantCultureIgnoreCase) > -1
```